spec/models/queue_item_spec.rb

```diff
@@ -29,6 +30,29 @@
     end
   end
   
+  describe "#rating=" do
+    let(:video) {Fabricate(:video)}
+    let(:user) {Fabricate(:user)}
+    let(:item) {Fabricate(:queue_item, user: user, video: video)}
+    
+    it "changes the rating if review is present" do
+      review = Fabricate(:review, user: user, video: video, rating: 2)
+      item.rating = 4
+      expect(Review.first.rating).to eq(4)
```

Comment:
I would be more explicit here as well. You have the `review` object, assert against it to be sure the code does what you want. Make sure you use `reload`.